src/Core/AndroidNative/maui/src/main/java/com/microsoft/maui/PlatformWrapperView.java

```diff
@@ -1,33 +1,134 @@
 package com.microsoft.maui;
 
+import android.app.Application;
 import android.content.Context;
+import android.os.Build;
+
+import android.graphics.BlurMaskFilter;
+import android.graphics.Color;
+import android.graphics.Bitmap;
+import android.graphics.drawable.Drawable;
 import android.graphics.Canvas;
+import android.graphics.Paint;
+import android.graphics.Path;
+import android.graphics.PorterDuff;
 import android.graphics.Rect;
+import android.graphics.Shader;
+
 import android.view.View;
 
 import androidx.annotation.NonNull;
 
+import com.microsoft.maui.PlatformPaintType;
+import com.microsoft.maui.PlatformShadowDrawable;
+import com.microsoft.maui.glide.ShadowBitmapPool;
+
+import com.bumptech.glide.load.engine.bitmap_recycle.BitmapPool;
+import com.bumptech.glide.load.engine.cache.MemorySizeCalculator;
+
 public abstract class PlatformWrapperView extends PlatformContentViewGroup {
+
     public PlatformWrapperView(Context context) {
         super(context);
         this.viewBounds = new Rect();
+        this.bitmapPool = ShadowBitmapPool.get(context);
```

Comment:
So, we use 1 pool per `PlatformWrapperView`.

Can we use 1 pool for the entire application? That would allow many views to share the pool.

Reply:
Actually that singleton method returns one pool for the entire application, so exactly what you're asking :)